packages/embark/src/lib/modules/blockchain_connector/index.js

```diff
@@ -420,10 +421,11 @@ class BlockchainConnector {
       '/embark-api/blockchain/transactions/:hash',
```

Comment:
Are you sure this API endpoint is not used by something else? Because the API is just `transactions/:hash` and not `transaction/:rawHash`